### DISCUSSION TITLE
Should RuntimeIdentifiers be updated to win-x instead of win7-x?

### DISCUSSION THREAD

**Discussion by mol** [cefsharp/CefSharp#5020] [General]
In [CefSharp.BrowserSubprocess.csproj](https://github.com/cefsharp/CefSharp/blob/master/CefSharp.BrowserSubprocess/CefSharp.BrowserSubprocess.csproj#L12) the RuntimeIdentifiers are listed as `<RuntimeIdentifiers>win7-x86;win7-x64</RuntimeIdentifiers>`

This caused some issues with code I have that is doing a x64 build, because the default then is to use a `win-x64` identifier.

There are other projects with the same identifiers. Perhaps those should all be updated to `<RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>`, when Windows 7 is anyway no longer supported?

**Comment by amaitland** [comment]
I don't remember if there was a specific reason for this, quick look at the change log and nothing stands out.

Changing from win 7 makes sense. Long as nothing breaks happy to switch to the more generic identifier.

**Reply by mol** [reply]
Should I go ahead and create a pull request where I make this change for all projects?

**Reply by amaitland** [reply]
PR would be welcome. Otherwise I can have a look at what's required next week.

**Reply by amaitland** [reply]
Done in commits https://github.com/cefsharp/CefSharp/commit/1b5dd7807d2e34b32080891c3aec60fe6375f036 and https://github.com/cefsharp/CefSharp/commit/2bf01fab6d8caf6d36065588204f1bd4d6571cb4

**Reply by mol** [reply]
Oh, cool - thanks. I was otherwise going to look at it this weekend and create a PR :)
